lint/importShadow_checker.go

```diff
@@ -0,0 +1,61 @@
+package lint
+
+//! Detects when imported package names shadowed in assignments.
+//
+// @Before:
+// import (
+// 	   "fmt"
```

Comment:
Looks like there is an extra whitespace here.

Reply:
yeap, but extra white space will need some fix in docs generation, will look what is going on there

Reply:
let's create additional issue for that, it seems like current approach with
```
for *index < len(lines) && len(lines[*index]) > 0 {
```
for finding description parts will not work for code with empty lines

Reply:
Its how our docs works. Empty lines in code are not allowed.

Reply:
This is mostly because we want:
1. Encourage minimal examples.
2. Avoid hard to read doc comments inside checkers source files.

I would use example like this:
```go
// "math" and "fmt" packages are imported.
func f(math int) int {
  fmt := math+1
  return fmt
}
```
Or even something that does not include surrounding func.